x-pack/plugins/ml/server/routes/datafeeds.ts

```diff
@@ -216,6 +234,9 @@ export function dataFeedRoutes({ router, mlLicense }: RouteInitialization) {
         params: datafeedIdSchema,
         query: deleteDatafeedQuerySchema,
       },
+      options: {
+        tags: ['access:ml:canDeleteJob'],
```

Comment:
See above. If we were to add a `canCreateDatafeed` then we would need a new `canDeleteDatafeed` capability.